config/dev/config.yaml

```diff
@@ -0,0 +1,12 @@
+project_code: sage-bionetworks
```

Comment:
sceptre configs support inheritance which means this config will inherit any configs from config/config.yaml so you don't need to repeat everything in this config.  You only need to append additional params specific to dev env.  You'll also want to add a config/prod/config.yaml file as well.